napalm_yang/mappings/junos/parsers/config/openconfig-network-instance/includes/ospfv2.yaml

```diff
@@ -1,39 +1,67 @@
 ---
-_process: not_implemented
+_process: unnecessary
 areas:
-    _process: not_implemented
+    _process: unnecessary
     area:
-        _process: not_implemented
+        _process:
+          - path: area
+            key: name
         config:
-            _process: not_implemented
+            _process: unnecessary
             identifier:
-                _process: not_implemented
+                _process:
+                  - pre: "{{ area_key }}"
         identifier:
-            _process: not_implemented
+            _process: unnecessary
         interfaces:
-            _process: not_implemented
+            _process: unnecessary
             interface:
-                _process: not_implemented
+                _process:
+                  - path: interface
+                    key: name
                 config:
-                    _process: not_implemented
+                    _process: unnecessary
                     authentication-type:
-                        _process: not_implemented
+                        _process:
+                          - path: authentication.md5.name
+                            post: "md5"
+                          - path: "authentication.simple-password"
+                            post: "simple-password"
+                          - path: authentication
+                            present: false
+                            post: "none"
```

Comment:
should this be null instead? I'd expect this leaf to be empty if there is no authentication. The model is unclear though here, it just says "string".

Reply:
ok, merging :)